lib/plugins/auto-load.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const _ = require('lodash');
+const debug = require('debug')('voxa');
+
+function autoLoad(skill, adapter, config) {
+  skill.onRequestStarted((alexaEvent) => {
+    if (!config) {
+      config = { loadByToken: false };
+    }
+
+    if (config.loadByToken && !_.get(alexaEvent, 'session.user.accessToken')) {
+      debug('Missing token on load data by token operation');
+      return alexaEvent;
+    }
+
+    let key = _.get(alexaEvent, 'session.user.userId') || _.get(alexaEvent, 'lambdaContext.System.user.userId');
```

Comment:
you should just be using http://voxa.readthedocs.io/en/latest/alexa-event.html#AlexaEvent.user

Also, `lambdaContext` has the information from `context` in the `(event, context, callback)` lambda handler, The context from the handler event is in 'alexaEvent.context.System.userId.